website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4037,6 +4156,29 @@ The output schema of this is:
   </tbody>
 </table>
 <br/>
+{{< /paragraph >}}
+
+{{< paragraph class="language-py" >}}
+<table>
+  <thead>
+    <tr class="header">
+      <th><b>Field Name</b></th>
+      <th><b>Field Type</b></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>key</td>
+      <td>ROW{userId:STRING, bank:STRING}</td>
+    </tr>
+    <tr>
+      <td>values</td>
+      <td>ITERABLE[ROW[Purchase]]</td>
+    </tr>
+  </tbody>
+</table>
+<br/>
+{{< /paragraph >}}
```

Comment:
It looks like our paragraph shortcode doesn't work when it has a table in it, both output schemas are shown:
![image](https://user-images.githubusercontent.com/675055/198152209-9dc9785e-7767-4f41-98d5-b9992b91b12d.png)

How about we just update the Java example to use "bank" instead of the nested field? Then we can use the same output schema for both.

Reply:
okay i'll update the java example to use bank and make appropriate changes for another table below